src/components/ImportExport.js

```diff
@@ -115,22 +150,36 @@ class ImportExport extends React.Component {
 
   exportFile = () => {
     const { trails, hydrants } = this.props;
-    const { selectedExport } = this.state;
+    const { selectedExport, exportType } = this.state;
+
 
     const trailFeatures = _.flatMap(_.values(trails.toJS()), item => item.features);
-    const hydrantFeatures = _.flatMap(_.values(hydrants.toJS()),item => item.features);
+    const hydrantFeatures = _.flatMap(_.values(hydrants.toJS()), item => item.feature);
+    const ext = exportType === 'GJ' ? 'json' : 'kml';
+
+    hydrantFeatures.forEach((f,i) => {
+      const desc = `${f.get('trailName')},${i},${f.get('name')}`
```

Comment:
Right now this won't actually work for new hydrants or ones associated to different trails. The only time `trailName` is assigned on a feature is when we import it with one (and that's how we associate the trails and hydrants from the different files). 

To work with his existing system, we should actually update this when the trail changes on a hydrant (or when it's created). I'm sure it's something he'll notice when he starts testing it out.